src/main/java/com/gladguys/alucoapi/repositories/customs/impl/CustomClassRepositoryImpl.java

```diff
@@ -43,4 +43,12 @@ public void deleteStudentFromClass(Long studentId, Long classId, Set<Long> exams
 		this.jdbcTemplate.update("DELETE FROM student_class WHERE student_id = ? AND class_id = ?", new Object[]{studentId, classId});
 	}
 
+	@Override
+	public boolean isClassFromTeacher(Long classId, Long teacherId) {
+		String sql = "SELECT count(*) FROM class WHERE teacher_id = ? AND id = ?";
```

Comment:
Uma dúvida, qual o sentido desse método? É para apenas saber se o teacher ensina naquela class? Se for, não seria melhor chamar o existsClassByTeacherId?